tagstudio/src/core/constants.py

```diff
@@ -107,6 +112,7 @@
 ]
 PROGRAM_TYPES: list[str] = [".exe", ".app"]
 SHORTCUT_TYPES: list[str] = [".lnk", ".desktop", ".url"]
+FONT_TYPES: list[str] = [".ttf", ".otf", ".woff"]
```

Comment:
Add `.ttc` and `.woff2` file extensions to the constants list (I haven't personally verified `.woff2` but `.ttc` seems to work fine)
```suggestion
FONT_TYPES: list[str] = [".ttf", ".ttc", ".otf", ".woff", ".woff2"]
```